crypto/rsa/rsa_pmeth.c

```diff
@@ -585,6 +595,14 @@ static int pkey_rsa_ctrl(EVP_PKEY_CTX *ctx, int type, int p1, void *p2)
         *(unsigned char **)p2 = rctx->oaep_label;
         return rctx->oaep_labellen;
 
+    case EVP_PKEY_CTRL_RSA_IMPLICIT_REJECTION:
+        if (rctx->pad_mode != RSA_PKCS1_PADDING) {
```

Comment:
This forces an order on setting various items - i.e. padding mode must be set before setting implicit rejection.
I don't see a value it doing that as this control is pretty clear what it is for - so it could reasonably be set before a padding mode is selected.

Reply:
Consistency with other settings is a reasonable argument.